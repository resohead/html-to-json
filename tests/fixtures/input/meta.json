[
    { "charset": "utf-8" },
    {
        "name": "viewport",
        "content": "width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0"
    },
    { "http-equiv": "x-ua-compatible", "content": "IE=edge,chrome=1" },
    { "http-equiv": "X-UA-Compatible", "content": "IE=11" },
    { "name": "author", "content": "Author Name Here" },
    { "name": "description", "content": "This is a test meta description" },
    { "name": "keywords", "content": "HTML, CSS, JavaScript" },
    { "name": "twitter:card", "content": "twitter-card" },
    { "name": "twitter:site", "content": "@organisation" },
    { "name": "twitter:title", "content": "Example Site Twitter Meta" },
    {
        "name": "twitter:description",
        "content": "This is a test og twitter description tag"
    },
    { "name": "twitter:creator", "content": "@handle" },
    {
        "name": "twitter:image",
        "content": "https://example.com/twitter-social-image.jpg"
    },
    { "property": "og:title", "content": "Example Site Title Meta" },
    { "property": "og:type", "content": "article" },
    { "property": "og:url", "content": "https://example.com" },
    {
        "property": "og:image",
        "content": "https://example.com/social-image-1.jpg"
    },
    {
        "property": "og:image",
        "content": "https://example.com/social-image-2.jpg"
    },
    {
        "property": "og:description",
        "content": "This is a test og description tag"
    },
    { "property": "og:site_name", "content": "Example Site Name Meta" }
]
