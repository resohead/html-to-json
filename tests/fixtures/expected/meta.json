{
    "hybrid": {
        "url": "https://example.com",
        "handle": "@organisation",
        "title": "Example Site Title Meta",
        "name": "Example Site Name Meta",
        "description": "This is a test meta description",
        "type": "article",
        "images": [
            "https://example.com/social-image-1.jpg",
            "https://example.com/social-image-2.jpg",
            "https://example.com/twitter-social-image.jpg"
        ],
        "image": "https://example.com/social-image-2.jpg",
        "author": "Author Name Here",
        "authorHandle": "@handle",
        "keywords": "HTML, CSS, JavaScript"
    },
    "author": "Author Name Here",
    "description": "This is a test meta description",
    "keywords": "HTML, CSS, JavaScript",
    "openGraph": {
        "images": [
            "https://example.com/social-image-1.jpg",
            "https://example.com/social-image-2.jpg"
        ],
        "title": "Example Site Title Meta",
        "type": "article",
        "url": "https://example.com",
        "image": "https://example.com/social-image-2.jpg",
        "description": "This is a test og description tag",
        "siteName": "Example Site Name Meta"
    },
    "twitter": {
        "images": ["https://example.com/twitter-social-image.jpg"],
        "card": "twitter-card",
        "site": "@organisation",
        "title": "Example Site Twitter Meta",
        "description": "This is a test og twitter description tag",
        "creator": "@handle",
        "image": "https://example.com/twitter-social-image.jpg"
    }
}
